internal/timeline/interval.go

```diff
@@ -0,0 +1,33 @@
+package timeline
+
+import "fmt"
+
+// interval.go
+type TIME float64
```

Comment:
Please refer to the naming conventions of other types in the project.

Reply:
TIME will be removed